README.md

```diff
@@ -291,10 +291,10 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
 ## W3C Verifiable Credential Data Model 1.1
 | Feature             | Typescript | Kotlin | Rust | Swift |
 | ------------------- | ---------- | ------ | ---- | ----- |
-| Creation            | ❌          | ❌      | ❌    | ❌     |
-| Signing as `vc-jwt` | ❌          | ❌      | ❌    | ❌     |
-| Verification        | ❌          | ❌      | ❌    | ❌     |
-| Validation          | ❌          | ❌      | ❌    | ❌     |
+| Creation            | ✅          | ✅      | ❌    | ❌     |
+| Signing as `vc-jwt` | ✅          | ✅      | ❌    | ❌     |
+| Verification        | ✅          | ✅      | ❌    | ❌     |
```

Comment:
is verification also as vc-jwt? wondering if we can change the header to be VC-JWT

Reply:
yeah think it should all just be vc-jwt